atd-vze/src/views/Crashes/Crashes.js

```diff
@@ -55,7 +55,7 @@ const Crashes = () => {
       minDate={minDate}
       roleSpecificColumns={adminColumns}
       hasSpecificRole={isAdminOrItSupervisor}
-      defaultSearchField="crash_id"
+      defaultSearchField="record_locator"
```

Comment:
i'm glad you changed this, although i'm not quite sure what effect this is having. i can open a follow-up issue to make whatever changes are needed to get temp records into the crashes list

Reply:
i had to update this to `record_locator` otherwise the page breaks because we arent using `crash_id` in the `crashGridTableParameters` anymore. what this does is set the field `record_locator` from the crash grid table parameters to be the default field being searched in the search bar when the page loads. though the label in the search bar still says "Search by crash id"